azurerm/internal/services/logic/logic_app_workflow_data_source.go

```diff
@@ -134,6 +139,12 @@ func dataSourceArmLogicAppWorkflowRead(d *schema.ResourceData, meta interface{})
 				d.Set("workflow_version", v["contentVersion"].(string))
 			}
 		}
+
+		var integrationAcc string
+		if props.IntegrationAccount != nil && props.IntegrationAccount.ID != nil {
+			integrationAcc = *props.IntegrationAccount.ID
```

Comment:
I believe we can just call d.Set("logic_app_integration_account_id", *props.IntegrationAccount.ID) here instead

Reply:
Same reason above.